packages/x-date-pickers-pro/src/internal/hooks/useDesktopRangePicker/useDesktopRangePicker.types.ts

```diff
@@ -26,7 +26,12 @@ export interface UseDesktopRangePickerSlotsComponent<TDate, TView extends DateOr
   Field: React.ElementType;
   FieldRoot?: React.ElementType<StackProps>;
   FieldSeparator?: React.ElementType<TypographyProps>;
-  Input?: React.ElementType<TextFieldProps>;
+  /**
+   * Component rendering an HTML input and a label inside the smart field.
```

Comment:
```suggestion
   * Component rendering an HTML input and a label inside the default field.
```

When reading that it let me think that the most important aspect is that it's the default ones. If they use a custom field and do not use `components` then it does not matter

Reply:
Yeah, I'd also be cautions about using this `smart` word... It's probably the only place we have this description for the field inputs 🙈